frontend/package.json

```diff
@@ -85,6 +85,7 @@
     "object-hash": "3.0.0",
     "ramda": "0.28.0",
     "react-error-boundary": "3.1.4",
+    "react-highlight": "^0.15.0",
```

Comment:
I'm hesitant to use react-highlight because it hasn't been updated in 2 years and it has a lot of open issues and PRs. So to me, the project looks abandoned. Could you consider using highlight.js directly instead?